DVLSForLinux/install-dvls.ps1

```diff
@@ -0,0 +1,435 @@
+    'DVLSPath'
+    'DVLSUser'
+    'DVLSGroup'
+    'DVLSAdminUsername'
+    'DVLSAdminPassword'
+    'DVLSAdminEmail'
+    'EnableTelemetry'
+    'DVLSCertificate'
+    'CreateDatabase'
+    'DatabaseHost'
+    'DatabaseUsername'
+    'DatabaseName'
+    @{
+        'Name'       = 'DatabaseEncryptedConnection'
+        'Expression' = {
+            if ($PSItem.DatabaseEncryptedConnection -Eq $Null) {
+                'Undefined'
+            }
+            else {
+                $PSItem.DatabaseEncryptedConnection
+            }
+        }
+    }
+    @{
+        'Name'       = 'DatabaseTrustServerCertificate'
+        'Expression' = {
+            if ($PSItem.DatabaseTrustServerCertificate -Eq $Null) {
+                'Undefined'
+            }
+            else {
+                $PSItem.DatabaseTrustServerCertificate
+            }
+        }
+    }
+    @{
+        'Name'       = 'ZipFile'
+        'Expression' = {
+            If ($PSItem.ZipFile -EQ $Null) {
+                'Undefined'
+            } Else {
+                $PSItem.ZipFile
+            }
+        }
+    }
+) | Format-List
+
+if ($Confirm) {
+    Write-Warning "If the above values look correct, enter [Y] or press Enter to continue" -WarningAction Inquire
+}
+
+Write-Verbose ("[{0}] Requesting 'sudo -v' for cached credentials" -F (Get-Date -Format "yyyyMMddHHmmss"))
+& sudo -v
+
+if (-Not [Bool](Get-Module -ListAvailable -Name 'Devolutions.PowerShell')) {
+    Write-Host ("[{0}] Installing Devolutions.PowerShell module for all users" -F (Get-Date -Format "yyyyMMddHHmmss")) -ForegroundColor Green
+
+    & sudo $PwshExecutable -Command {
+        Install-Module -Name 'Devolutions.PowerShell' -Confirm:$False -Scope 'AllUsers' -Force
+    }
+}
+
+try {
+    Import-Module -Name 'Devolutions.PowerShell' -Scope 'Global' -ErrorAction 'Stop'
+}
+catch {
+    Write-Error ("[{0}] The Devolutions.PowerShell module failed to load, aborting installation" -F (Get-Date -Format "yyyyMMddHHmmss"))
+    Exit
+}
+
+if (-Not [Bool](Get-Module -Name 'Devolutions.PowerShell')) {
+    Write-Error ("[{0}] The Devolutions.PowerShell module failed to install, aborting installation" -F (Get-Date -Format "yyyyMMddHHmmss"))
+    Exit
+}
+
+Write-Host ("[{0}] Creating user ({1}), group ({2}), and directory ({3})" -F (Get-Date -Format "yyyyMMddHHmmss"), $DVLSVariables.DVLSUser, $DVLSVariables.DVLSGroup, $DVLSVariables.DVLSPath) -ForegroundColor Green
+
+& sudo $PwshExecutable -Command {
+    Param(
+        $DVLSVariables
+    )
+
+    & useradd -N $DVLSVariables.DVLSUser
+    & groupadd $DVLSVariables.DVLSGroup
+    & usermod -a -G $DVLSVariables.DVLSGroup $DVLSVariables.DVLSUser
+    & usermod -a -G $DVLSVariables.DVLSGroup $DVLSVariables.CurrentUser
+    & mkdir -p $DVLSVariables.DVLSPath
+    & chown -R ("{0}:{1}" -F $DVLSVariables.DVLSUser, $DVLSVariables.DVLSGroup) ([System.IO.DirectoryInfo]$DVLSVariables.DVLSPath).Parent
+    & chmod 770 ([System.IO.DirectoryInfo]$DVLSVariables.DVLSPath).Parent
```

Comment:
I might be missing something, but why is the .Parent part there? My undertanding here is that, if you install it in /opt/dvls, you've just changed the permissions to /opt/ which is very bad